src/index.js

```diff
@@ -92,14 +92,19 @@ function download (version, platform, arch, installPath) {
     process.stdout.write(`Downloading ${url}\n`)
 
     request.get(url, (err, res, body) => {
+      if (err) {
+        // TODO handle error: haad?
+      }
       // Handle errors
-      if (res.statusCode !== 200)
+      if (res.statusCode !== 200) {
         error(new Error(`${res.statusCode} - ${res.body}`), reject)
```

Comment:
@haadcode I can't find this `error` function, maybe it got lost somewhere on your PR?? Could you check